controllers/links/user.js

```diff
@@ -1,26 +1,44 @@
 const {FirebaseDynamicLinks} = require('firebase-dynamic-links');
+const UsersFunction = require('../../databases/functions/users');
+const {User} = require('../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you're importing the entire `User` model just to pass it as an argument to `findUserByUsername`. If `findUserByUsername` is always going to work with the `User` model, consider importing it directly within the `UsersFunction` module to reduce unnecessary dependencies and make the code cleaner.
```diff
- const UsersFunction = require('../../databases/functions/users');
- const {User} = require('../../databases/models');
+ const UsersFunction = require('../../databases/functions/users');
```
And in `UsersFunction`:
```diff
- async function findUserByUsername(model, username) {
+ const {User} = require('../models');
+ async function findUserByUsername(username) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->